### DISCUSSION TITLE
Help!! with adding overlays to live streams

### DISCUSSION THREAD

**Discussion by FranciscoNetoBr** [ant-media/Ant-Media-Server#6262] [Q&A] [answered]
Good morning everyone, first of all, I'd like to thank this community, which has always been very attentive, helpful, and especially patient with me.

I need some help that will be a game-changer in my old project.

I need to integrate overlays with ads and important information for my community who follows me on my 'infinite' live stream on YouTube, if possible controlled by Uno (https://overlays.uno).

- Firstly, I'm not sure where to start and if it's possible. I saw something at the following address: https://streamtoolbox.com/stamp/.
- Since it's a stream spanning months, I don't want to have to restart it by restarting the server or the live stream for now.
- I saw something about a plugin, but I'm not sure if it meets my needs.
- Lastly, if it's possible and there's a solution, I'd appreciate it if you could provide step-by-step guidance, as I have limitations in knowledge.

Information: I'm using AMS  Community Edition 2.6.0 20230517_0515 on an Oracle VPS.

(I'm just letting you know that I'm Brazilian and I use the translator to communicate with you, so forgive me if something isn't right.)

**Comment by Mohit-3196** [comment]
Hi @FranciscoNetoBr,
How are you?

It's great to know that you have been able to get support from the community and it's helping you proceed :)

There are some ways to add the overlays to the stream and for that, you can make use of the [filter plugin](https://antmedia.io/discover-the-filter-plugin/)
Alternatively, you can also use the [stamp plugin](https://antmedia.io/marketplace/stamp-app/) as you mentioned above.

The important thing to note here is that these functions and plugins can be carried out on the Ant Media Server Enterprise edition only and not the community edition.
So, you can get the enterprise edition from the [antmedia.io](https://antmedia.io/) or you can even launch it from marketplaces like AWS, Azure, GCP, DO, etc.

After you get the enterprise edition, you can follow the filter plugin document or get the stamp plugin from the AMS marketplace and run it.

I hope it helps.
Thank you.

**Reply by FranciscoNetoBr** [reply]
Mohit-3196,

I really appreciate your attention and consideration, yes, I feel good and comfortable being here with you and answering my questions.

As I said, I am Brazilian and my projects so far do not generate income for me, and the dollar is very expensive here for us Brazilians.

This makes it impossible to obtain Ant Media Server Enterprise. Which limits me to only using the community version.

These overlaps would help me a lot, even financially, as it would generate more interaction with the public.

Unfortunately, we Brazilians do not have access to many of the facilities that you have.

Thanks again for your help.

**Reply by Mohit-3196** [reply]
Hi @FranciscoNetoBr,
No worries. 
Please reach out to our solutions team at contact@antmedia.io and mention this discussion thread. We can help you with some discounts and support on the enterprise edition license so that finance will not be an issue.

Thank you.

**Reply by FranciscoNetoBr** [reply]
Mohit-3196,

How are you? I hope you're doing well.

The delay in responding to your post was because I had already "given up" on going in this direction. Just imagine that the value of $99 in Brazil is equivalent to 45% of the minimum wage, so I was feeling hopeless.

I do live streams on YouTube channels, however, none of them are monetized, so I don't have any revenue yet.

The Ant Media Server (AMS) is a great solution for video streaming for me, and the AMS from the Community gives us a very important start; I am truly grateful for all of your support.

Following your guidance, I will indeed send an email to contact@antmedia.io, maybe we can find a way around this cost issue. Let's hope for the best.

Yes, once again, I can't thank you enough for your attention, always helpful. Thank you!!

**Reply by Mohit-3196** [reply, marked as answer]
Hi @FranciscoNetoBr,
No worries about the costs.

We will be more than happy to help you and see you succeed.
The team is on holiday this week so there might be some delay in the response but be rest assured, the solutions team will reach you.
Let me know after you write.

Thank you.

**Reply by FranciscoNetoBr** [reply]
My dear, I couldn't find words to thank you for your attention, I hope you receive double the kindness with which you treat me.

I will wait for the AMS team to contact you, and I will send you the results.

Thank you again for your generosity.

Yours sincerely

Francisco Neto

**Reply by Mohit-3196** [reply]
Hi @FranciscoNetoBr,
I hope you're doing well!

It's Mohit reaching out again 😊. I've been thinking about how we can support you and welcome you aboard our journey to becoming the streaming engine of the world 🌍.
We're happy to offer you a complimentary annual license for a year to help you move forward.

I trust this will aid you in reaching your goals. 😊
I shared the details with you via email as well.

Looking forward to hearing from you.

--
Best Regards,
Mohit Dubey

**Reply by FranciscoNetoBr** [reply]
Good Morning my friend Mohit Dubey,

It is truly a great pleasure to continue with your support in this project, I was truly surprised by the affection and attention given.

As I said in my email response, I couldn't find words to express my gratitude to you and the others who in some way contributed to getting us to this moment.

Well, now it's time to think about work and to take AMS to all corners of the world, first, I will follow the instructions given in the email.

Then I turn to your help and support regarding the change/configuration of the AMS Community that I am using.

There are many questions: One of the most important is; Is it possible to do all this without having to stop the service? Or install a new MAS Enterprises?

Best Regards,
Francisco Neto

**Reply by Mohit-3196** [reply]
Hi @FranciscoNetoBr,
Thank you for your words :)

I will recommend you make a fresh installation with the enterprise edition.
Thank you.

**Comment by mekya** [comment]
Hi,

Another option for overlays is using the MediaPush Plugin
https://github.com/ant-media/Plugins/tree/master/MediaPushPlugin

I'm more than happy to make a demo if anyone is interested. 

Regards
Oguz
